CONTRIBUTING.md

```diff
@@ -32,5 +32,5 @@ accept your pull requests.
    (https://github.com/GoogleCloudPlatform/Template/wiki/style.html) for the
    recommended coding standards for this organization.
 1. Ensure that your code has an appropriate set of unit tests which all pass.
-1. Submit a pull request.
+2. Submit a pull request.
```

Comment:
this is likely unnecessary. markdown auto increments ordered lists. just use `1.` everywhere.